### PR TITLE
[Bug 1182904] Gather plugin version/filename in AAQ

### PR DESCRIPTION
**Pull request by philipp-sumo**

### CODE REVIEW

File: kitsune/sumo/static/sumo/js/aaq.js [hunk 1]
```diff
@@ -107,9 +107,12 @@ AAQSystemInfo.prototype = {
...
         // Returns wiki markup for the list of plugins
         var plugins = [],
-            i, d;
+            i, strtemp, d;
         for (i = 0; i < navigator.plugins.length; i++) {
-            d = navigator.plugins[i].description.replace(/<[^>]+>/ig,'');
+            strtemp = navigator.plugins[i].name;
+            if (navigator.plugins[i].version) strtemp += ' ' + navigator.plugins[i].version;
+            if (navigator.plugins[i].filename) strtemp += ' (' + navigator.plugins[i].filename + ')';
+            d = strtemp.replace(/<[^>]+>/ig,'');
```
Comment (willkg) [inline kitsune/sumo/static/sumo/js/aaq.js]: Nit: Pretty sure there should be a space between the comma and the first quote.

**Comment by willkg** [comment]
Nit: The commit message doesn't conform to our guidelines. That's an issue since it won't work with our tools very well.

http://kitsune.readthedocs.org/en/latest/conventions.html#git-commit-messages

Can you write tests for this? Amongst other things, it'll help document the expectation of the shape of things you're parsing out for future maintainers.

File: kitsune/sumo/static/sumo/js/aaq.js (hunk 1 above)
Comment (philipp-sumo) [inline kitsune/sumo/static/sumo/js/aaq.js]: that part is copied over from the existing code, but i could change that of course if you like...

**Comment by philipp-sumo** [comment]
hi will, i did change the commit message accoridingly but i'm sorry that i can't write a test for this change, since i'm not knowledgeable enough for that...

**Comment by safwanrahman** [comment]
I think I can write test for this! I will try to write a test for this!
@willkg Can you please let me know, what the tests should do?

**Comment by willkg** [comment]
The tests should verify the code is working as intended. What kind of strings are being manipulated here? What kind of objects are being operated on? What possible error situations can occur?

**Comment by mythmon** [comment]
You can add tests for JS code in [kitsune/sumo/static/sumo/js/tests](https://github.com/mozilla/kitsune/tree/master/kitsune/sumo/static/sumo/js/tests).

**Comment by safwanrahman** [comment]
Sorry, have no Idea about Javascript tests! :disappointed:

**Comment by mythmon** [comment]
I don't want to merge this without tests. Unfortunately, the state of our JS tests aren't great, so this might be hard to accomplish. For now, I'm going to close this to clean things up. Let me know if you want to work on tests, and I can help you out.

**Comment by philipp-sumo** [comment]
i've asked around amongst contributors but noone of us will be able to write js tests for this patch - is this something that sumodev could do, since the change in the pull-request itself would be most useful in the support forum?

**Pull request closed without merging**